backend/inspirehep/records/api/literature.py

```diff
@@ -144,6 +144,9 @@ def update_refs_to_conferences(data):
                 continue
             conference["conference_record"] = get_ref_from_pid("con", pid)
 
+    def copy(self):
```

Comment:
you don't need this, using the parent method is the default if not defined on the subclass.